### PR TITLE
postgraphile website: PostGraphile Pro page

### PR DESCRIPTION
**Pull request by jemgillam**
## Description

- Adds fontawesome
- Adds columns for sponsorship
- Adds rows showing some sponsorship benefits
- Adds a PRO badge and a new SPON badge
- Updates FAQ wording to distinguish between V4 core, V4 Pro and V5 (original wording is on graphile.org)
- Wording up top is completely new, as discussed

WIP,  still needs:

- [x]  Linting
- [x] Mobile version
- [x] Navbar link
- [x] Check dark mode is OK

### CODE REVIEW

**Comment by jemgillam** [comment]
I've added two new columns and some rows at the bottom to help promote the sponsorship model of support. 

Screenshot 2024-02-15 at 11 07 16

Screenshot 2024-02-15 at 11 07 50

I've implemented the "pro" badge and added a "spon" one. At the moment, they are rarely used in the documentation but I hope to sprinkle the "spon" one a few other places eg in an info box about support or something. We will discuss this - I do not want to over do it and make it feel like pressurised selling, it only should be wherever it makes sense to add it. 

Screenshot 2024-02-15 at 11 09 15

**Comment by jemgillam** [comment]
A taste of dark-mode and mobile view:

Screenshot 2024-02-15 at 17 23 45

Screenshot 2024-02-15 at 17 23 56

File: postgraphile/website/versioned_docs/version-4.12.0/plugins.md [hunk 1]
```diff
@@ -54,7 +57,9 @@ set GRAPHILE_LICENSE="license_key_from_graphile_store" & postgraphile -c postgre
 $env:GRAPHILE_LICENSE="license_key_from_graphile_store"; postgraphile -c postgres://...
 ```
 
-**IMPORTANT**: these plugins do not "phone home" so you'll need to update your
+:::caution important
```
Comment (benjie) [inline postgraphile/website/versioned_docs/version-4.12.0/plugins.md:60]: Nice :+1:

File: postgraphile/website/versioned_docs/version-4.12.0/production.md [hunk 2]
```diff
@@ -234,12 +237,16 @@ you wish arbitrary third parties to be able to send queries to your API without
 the queries being pre-approved and without the risk of bringing your servers to
 their knees.
 
-**You are highly encouraged to purchase the [Pro Plugin [PRO]](/pricing/), which
-implements these protections in a deeply integrated and PostGraphile optimised
-way, and has the added benefit of helping sustain development and maintenance on
-the project.** You can read the
+:::info
```
Comment (benjie) [inline postgraphile/website/versioned_docs/version-4.12.0/production.md:240]: Maybe "tip" would be better?

**Review by benjie — APPROVED** [review]
LGTM! :raised_hands:

**Pull request merged**